Packs/CortexAttackSurfaceManagement/Playbooks/Cortex_ASM_-_Splunk_Enrichment_README.md

```diff
@@ -0,0 +1,44 @@
+Given the IP address this playbook enriches information from Splunk results relevant to ASM alerts. 
+
+## Dependencies
+
+This playbook uses the following sub-playbooks, integrations, and scripts.
+
+### Sub-playbooks
+
+This playbook does not use any sub-playbooks.
+
+### Integrations
+
+This playbook does not use any integrations.
+
+### Scripts
+
+* GetTime
+* GridFieldSetup
+
+### Commands
+
+* splunk-search
+
+## Playbook Inputs
+
+---
+
+| **Name** | **Description** | **Default Value** | **Required** |
+| --- | --- | --- | --- |
+| IP | Remote IP in an incident/alert.  | alert.remoteip | Required |
+
+## Playbook Outputs
+
+---
+
+| **Path** | **Description** | **Type** |
+| --- | --- | --- |
+| asmserviceowner | This is the potential service owners from splunk results. | unknown |
```

Comment:
I don't think this is necessary because the `GridFieldSetup` automation in the playbook it exporting the value to the field, therefore it doesn't need to returned as raw context.  What do you think?

Reply:
Agreed, that was leftover from a previous change. I'll remove it.

Reply:
Resolved in 92b7a41.